drivers/gpu/drm/panel/panel-nt36672-tianma.c

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2022 FIXME
+// Generated with linux-mdss-dsi-panel-driver-generator from vendor device tree:
+//   Copyright (c) 2013, The Linux Foundation. All rights reserved. (FIXME)
+
+#include <linux/delay.h>
+#include <linux/gpio/consumer.h>
+#include <linux/module.h>
+#include <linux/of.h>
+#include <linux/regulator/consumer.h>
+
+#include <video/mipi_display.h>
+
+#include <drm/drm_mipi_dsi.h>
+#include <drm/drm_modes.h>
+#include <drm/drm_panel.h>
+
+struct nt36672_tianma {
+	struct drm_panel panel;
+	struct mipi_dsi_device *dsi;
+	struct regulator *supply;
+	struct gpio_desc *reset_gpio;
+	bool prepared;
+};
+
+static inline struct nt36672_tianma *to_nt36672_tianma(struct drm_panel *panel)
+{
+	return container_of(panel, struct nt36672_tianma, panel);
+}
+
+#define dsi_dcs_write_seq(dsi, seq...) do {				\
```

Comment:
almost identical macro is already upstream, no need to reinvent a new one (since 6.1-rc, you still need it for <=6.0)

Reply:
So what exactly needs to be changed?
That was generated from the panel generator so I'm not entirely sure on what exactly is going on there.

Reply:
Remove this macro (`dsi_dcs_write_seq`) and use this one instead `[mipi_dsi_dcs_write_seq](https://elixir.bootlin.com/linux/latest/C/ident/mipi_dsi_dcs_write_seq)`

Reply:
However if you wish this deice appear in pmaport *right now* then you need to target this MR on 6.0.y branch and keep the macro

Reply:
Eh I can wait (still gotta do the device package properly), i'll look into the macro thing in a bit.

Reply:
The change should be simple replace `dsi_dcs_write_seq` to `mipi_dsi_dcs_write_seq` and removing declaration of `dsi_dcs_write_seq` completely.

> how do I update this MR

You push (or force-push) a commit to your branch